### PR TITLE
Register thisiskeanyvy.is-a.dev

### PR DESCRIPTION
**Pull request by thisiskeanyvy**
Added `thisiskeanyvy.is-a.dev` using the [CLI](https://www.npmjs.com/package/@is-a-dev/cli).

### CODE REVIEW

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please use a valid record.

**Comment by thisiskeanyvy** [comment]
- As far as I'm concerned, it's all good.

Many thanks in advance!

CNAME

File: domains/thisiskeanyvy.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "thisiskeanyvy",
+        "email": "notifkeanyvy@gmail.com"
+    },
+
+    "record": {
+        "CNAME": ["thisiskeanyvy.pages.dev"]
```
Comment (VaibhavSys) [inline domains/thisiskeanyvy.json:8]: CNAME key must have a string value, it cannot be an array.

**Review by VaibhavSys — CHANGES_REQUESTED** [review]

File: domains/thisiskeanyvy.json (hunk 1 above)
Comment (thisiskeanyvy) [reply domains/thisiskeanyvy.json]: OK, thank you very much!

**Review by VaibhavSys — APPROVED** [review]

**Review by wdhdev — APPROVED** [review]

**Pull request merged**